examples/bert/nv_user_script.py

```diff
@@ -0,0 +1,30 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import torch
+from datasets import load_dataset  # type: ignore[import]
+from datasets.utils import logging as datasets_logging  # type: ignore[import]
+from transformers import AutoTokenizer, BertModel  # type: ignore[import]
+
+datasets_logging.disable_progress_bar()
+datasets_logging.set_verbosity_error()
+
+
+def load_pytorch_origin_model():
```

Comment:
have tested this example locally? the model_loader function is always called with `model_path` whether it is none or not

Reply:
you can empty the cache directory when rerunning after making code changes. This can also be done by adding `"clean_cache": true` under `"engine"` section during dev. otherwise, it will just reload from cache if the config json did not change.

Reply:
I tested locally, it works.

Reply:
Sorry, there was a formatting error. Please start the pipeline one more time.